src/Semantic/CLI.hs

```diff
@@ -1,10 +1,12 @@
-{-# LANGUAGE ApplicativeDo, FlexibleContexts #-}
+{-# LANGUAGE ApplicativeDo #-}
+{-# LANGUAGE FlexibleContexts #-}
 module Semantic.CLI (main) where
 
+import qualified Analysis.File as File
```

Comment:
Pretty much convinced `Analysis.File` belongs in `semantic-source`, but again, that does not need to block this PR.